hummingbot/connector/exchange/loopring/loopring_exchange.pyx

```diff
@@ -479,9 +479,10 @@ cdef class LoopringExchange(ExchangeBase):
             res = await self.api_request("DELETE", ORDER_CANCEL_ROUTE, params=cancellation_payload, secure=True)
             code = res['resultInfo']['code']
             message = res['resultInfo']['message']
-            if code == 102117:
-                # Order didn't exist on exchange, mark this as canceled
+            if code == 102117 and in_flight_order.created_at < (int(time.time()) - UNRECOGNIZED_ORDER_DEBOUCE):
+                # Order doesn't exist and enough time has passed so we are safe to mark this as canceled
                 self.c_trigger_event(ORDER_CANCELLED_EVENT, cancellation_event)
+                self.stop_tracking(tracked_order.client_order_id)
```

Comment:
This should be `self.stop_tracking(client_order_id)`

Reply:
These comments should be addressed, but revealed another issue where creation events were issued after order failures were issued. That new issue is fixed in this pr as well.